source/guides/network/running-a-symbol-node.rst

```diff
@@ -230,7 +230,7 @@ Just like in the harvesting case, this creates the required :ref:`votingkeylinkt
       symbol-bootstrap updateVotingKeys
```

Comment:
Should a guide to rewnew certificates with bootstrap be added? Probably in a new PR.

https://github.com/fboucquez/symbol-bootstrap/blob/dev/docs/renewCertificates.md

Reply:
Yes, renew certificates is a new PR.  Can you write one?